<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrearVariacion</class>
 <widget class="QWidget" name="CrearVariacion">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>459</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Registro Presupuesto solocitado</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/iconos/cosas.jpg</normaloff>:/iconos/cosas.jpg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Variación de Gastos por Unidad Presupuestaria Linea de Trabajo,Rubro, Cuenta y Especifico</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Datos Generales de Variación de Gastos por Unidad  Presupuestaria</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="Institucion">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Institucion:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="EjercicioFiscal">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Ejercicio Fiscal:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="EjercicioFiscal2">
             <property name="minimum">
              <number>2010</number>
             </property>
             <property name="maximum">
              <number>2999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="4">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QSplitter" name="splitter_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="EjercicioFiscal_4">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Fuente Financiamiento:</string>
              </property>
             </widget>
             <widget class="QComboBox" name="comboBox">
              <property name="editable">
               <bool>false</bool>
              </property>
              <property name="duplicatesEnabled">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>0201 Fondo General</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>0101 Fondos Propios</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>0203 Donado</string>
               </property>
              </item>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="5">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="FechaElaboracion">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Fecha Elaboracion</string>
              </property>
             </widget>
             <widget class="QDateEdit" name="dateElaboracion">
              <property name="dateTime">
               <datetime>
                <hour>0</hour>
                <minute>0</minute>
                <second>0</second>
                <year>2009</year>
                <month>11</month>
                <day>8</day>
               </datetime>
              </property>
              <property name="minimumDateTime">
               <datetime>
                <hour>0</hour>
                <minute>0</minute>
                <second>0</second>
                <year>2009</year>
                <month>11</month>
                <day>8</day>
               </datetime>
              </property>
              <property name="minimumDate">
               <date>
                <year>2009</year>
                <month>11</month>
                <day>8</day>
               </date>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
              <property name="date">
               <date>
                <year>2009</year>
                <month>11</month>
                <day>8</day>
               </date>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="Institucion2">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Especificacion Presupuesto Solicitado</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTableWidget" name="tablaPresupuestoSolicitado">
           <property name="minimumSize">
            <size>
             <width>261</width>
             <height>211</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>600</width>
             <height>500</height>
            </size>
           </property>
           <property name="cursor" stdset="0">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked</set>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectItems</enum>
           </property>
           <property name="textElideMode">
            <enum>Qt::ElideNone</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerItem</enum>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="rowCount">
            <number>0</number>
           </property>
           <property name="columnCount">
            <number>3</number>
           </property>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>30</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>20</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>30</number>
           </attribute>
           <column>
            <property name="text">
             <string>   Codigo Cuenta</string>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string notr="true">Nombre de Cuenta</string>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Presupuesto Solicitado</string>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </column>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Registro: Presupuesto Solicitado</string>
           </property>
           <widget class="QWidget" name="layoutWidget_2">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>80</y>
              <width>111</width>
              <height>91</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="AgregarButton">
               <property name="text">
                <string>Agregar</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="CancelarButton">
               <property name="text">
                <string>Cancelar</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/iconos/pre2-iconos.png</normaloff>:/iconos/pre2-iconos.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="1" column="0" rowspan="2">
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>Total Supuesto Solicitado</string>
           </property>
           <widget class="QCommandLinkButton" name="obtenerSolicitadoLinkButton">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>20</y>
              <width>221</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Obtener Total Solicitado</string>
            </property>
           </widget>
           <widget class="QSplitter" name="splitter_2">
            <property name="geometry">
             <rect>
              <x>410</x>
              <y>30</y>
              <width>139</width>
              <height>20</height>
             </rect>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QLineEdit" name="montoTotallLine">
             <property name="text">
              <string/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Opciones</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="salirButton">
              <property name="text">
               <string>Salir</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>salirButton</sender>
   <signal>clicked()</signal>
   <receiver>CrearVariacion</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>946</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
